### DISCUSSION TITLE
504 gateway timeout

### DISCUSSION THREAD

**Discussion by Nirbles** [chuot/rdio-scanner#207] [Q&A]
I have Windows 10. SDRTrunk dongle which is working fine. Rdio Scanner was working fine but after reboot Rdio Scanner says no link. Dirwatch is set but keeps giving a 504 gateway timeout error. Everything was fine before reboot. Any help appreciated!

**Comment by chuot** [comment]
It looks like the rdio-scanner process isn't started. Have you installed it as a service or were you running it at the command line? It is recommended to install it as a service so it gets automatically restarted after reboots. Look in the PDF document for instructions in regards to running rdio-scanner as a service.

**Comment by Nirbles** [comment]
Thank you...I got an error message running that command that it couldnt find path but moved the setup files to that defined directory and restarted and its working so we'll see...Thanks again
